src/Common/formatIPv6.cpp

```diff
@@ -146,8 +146,9 @@ void formatIPv6(const unsigned char * src, char *& dst, uint8_t zeroed_tail_byte
             uint8_t ipv4_buffer[IPV4_BINARY_LENGTH] = {0};
             memcpy(ipv4_buffer, src + 12, IPV4_BINARY_LENGTH);
             // Due to historical reasons formatIPv4() takes ipv4 in BE format, but inside ipv6 we store it in LE-format.
+#if __BYTE_ORDER == __ORDER_LITTLE_ENDIAN__
```

Comment:
Let's use [C++ native syntax](https://en.cppreference.com/w/cpp/types/endian) instead of macros.